### ISSUE TITLE
Configuration parameter always tries to tab complete to default folder

### ISSUE THREAD

**Issue by ghost** [jdhitsolutions/WindowsSandboxTools#7]
Whenever I try to use tab completion with the -Configuration parameter it always completes to the default path

`Start-WindowsSandbox -Configuration `
I press TAB ->
`Start-WindowsSandbox -Configuration C:\Users\dominik\Documents\PowerShell\Modules\WindowsSandboxTools\1.1.0\wsbConfig\basic.wsb`

It's also ignoring whatever I already typed, like

`Start-WindowsSandbox -Configuration .\Sandbox\Configuration`
-> TAB
`Start-WindowsSandbox -Configuration C:\Users\dominik\Documents\PowerShell\Modules\WindowsSandboxTools\1.1.0\wsbConfig\basic.wsb`

I could reproduce the issue in PowerShell 5 and PowerShell 7.3.6

Cheers,
Dominik

**Comment by jdhitsolutions** [comment]
Do you have the `wsbConfigPath` variable defined?

**Comment by jdhitsolutions** [comment]
I just did a clean install in Windows PowerShell and verified this works.  It shouldn't make a difference but did you try manually importing the module first?

**Comment by ghost** [comment]
> Do you have the `wsbConfigPath` variable defined?

No, the variable is not defined, but when I define it, it auto-completes to the variable

> I just did a clean install in Windows PowerShell and verified this works. It shouldn't make a difference but did you try manually importing the module first?

I am not sure what you mean by manually importing. I just installed the module with

`Install-Module WindowsSandboxTools`

and that's it. :)

**Comment by jdhitsolutions** [comment]
That is odd. When you import the module, just as you are doing, it should automatically define $wsbConfigPath and $wsbScripts. These are used for autocompletion so if they get deleted, you'll see the behavior you described.

In a new PowerShell session, import the module then run `Get-Variable wsb*`. If you don't see the variables, that's a problem we have to figure out.

I installed the module on a new Windows 10 system and imported it with no problem.
